Safewaan/HonkedIn
# Users can view their comments on a separate page

## Trello ticket link

[As a user, I want to see my own comments on a separate page.](https://trello.com/c/MiVaTpdx/110-as-a-user-i-want-to-see-my-own-comments-on-a-seperate-page)


## Implementation description
* Users can view their own comments on a separate page called "My Comments" 


## Steps to test
1. Please go to [My Comments](http://localhost:3000/my-comments) 
2. Please ensure that you are able to see all of your comments on the page
3. Please ensure that you can click on the title to view the forum page in full detail 
4. Please ensure that you do not view others' comments on the page
5. Please also create a new comment and ensure that you can view the comment on the "My Comments" page


## What should reviewers focus on?
* Please ensure you see all of your own comments on the "My Comments"
